aten/src/ATen/gen.py

```diff
@@ -175,7 +175,7 @@ def check_all_files_written(self):
 }
 
 backends = ['CPU', 'CUDA']
-densities = ['Dense', 'Sparse']
+densities = ['Dense', 'Sparse', 'Mkldnn'] # TODO: layout instead of densities?
```

Comment:
Probably "densities" should be renamed to "layouts" here. Please suggest whether need to change in this PR or not. Thanks.